src/main/java/com/thoughtworks/forecastAlexaSkillBackend/model/ShoppingItem.java

```diff
@@ -0,0 +1,76 @@
+package com.thoughtworks.forecastAlexaSkillBackend.model;
+
+import com.google.gson.internal.$Gson$Types;
+
+public class ShoppingItem {
```

Comment:
```suggestion
@Getter
@Setter
public class ShoppingItem {
```

We can use the getter and setter annotations here. that will avoid all the methods that we need to write below for getting and setting

Reply:
done